src/rpc/mempool.cpp

```diff
@@ -669,6 +670,136 @@ static RPCHelpMan savemempool()
     };
 }
 
+static RPCHelpMan submitpackage()
+{
+    return RPCHelpMan{"submitpackage",
+        "\nSubmit a package of raw transactions (serialized, hex-encoded) to local node (-regtest only).\n"
+        "\nThe package will be validated and accepted to mempool if it passes consensus and mempool policy rules. Throws a JSONRPCError if anything goes wrong.\n"
+        "\nExperimental. API may be unstable. Refer to docs/policy/packages.md for documentation on package policies.\n"
+        "\nWarning: successful submission does not mean the transaction will propagate to other nodes on the network.\n",
+        {
+            {"package", RPCArg::Type::ARR, RPCArg::Optional::NO, "An array of raw transactions.\n",
+                {
+                    {"rawtx", RPCArg::Type::STR_HEX, RPCArg::Optional::OMITTED, ""},
+                },
+            },
+        },
+        RPCResult{
+            RPCResult::Type::OBJ, "", "",
+            {
+                {RPCResult::Type::OBJ_DYN, "tx-results", "transaction results keyed by wtxid",
+                {
+                    {RPCResult::Type::OBJ, "xxxx", "transaction wtxid", {
+                        {RPCResult::Type::STR_HEX, "txid", "The transaction hash in hex"},
+                        {RPCResult::Type::STR_HEX, "other-wtxid", /*optional=*/ true, "The wtxid of a different transaction with the same txid but different witness found in the mempool. This means the submitted transaction was ignored."},
+                        {RPCResult::Type::NUM, "vsize", "Virtual transaction size as defined in BIP 141."},
+                        {RPCResult::Type::OBJ, "fees", "Transaction fees (only present if result=\"valid\")", {
```

Comment:
```suggestion
                        {RPCResult::Type::OBJ, "fees", "Transaction fees", {
```

(result is always valid if this rpc returns)

Reply:
Ah, leftover from previous iteration, thanks for catching